src/middlewared/middlewared/plugins/system/product.py

```diff
@@ -98,6 +98,7 @@ def version_major(self):
     @accepts()
     @returns(Str('truenase_release_notes_url'))
```

Comment:
truenase ?

Reply:
yeah! it's the new "hip" way of spelling 😉 ....thanks fixed it